CHANGES/8258.removal

```diff
@@ -0,0 +1 @@
+Django is now patched in FIPS mode when pulp_devel role is specified
```

Comment:
I think this should be split into two release notes.

8258.removal: FIPS support is removed due to to Django (and dependency of Pulp) not being FIPS compatible.

8258.dev: The dev role patches Django to allow continued FIPS compatibility development within Pulp in preparation for Django to add FIPS support at some point.